### PR TITLE
feat: tenderdash index template

### PR DESCRIPTION
**Pull request by strophy**
This PR adds an index template and filebeat input for Tenderdash log data.

## Issue being fixed or feature implemented
Tenderdash JSON logs were lacking a timestamp in the past. This was reported and resolved [upstream](https://github.com/tendermint/tendermint/issues/6146). 

## What was done?
- Create index template
- Create filebeat input 

## How Has This Been Tested?
On Palinka

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

File: ansible/roles/elastic-beats/tasks/main.yml [hunk 1]
```diff
@@ -89,6 +93,28 @@
                     - json.hostname
                     - json.time
                   ignore_missing: true
+          - type: container
```
Comment (shumkov) [inline ansible/roles/elastic-beats/tasks/main.yml:96]: I'm afraid container approach won't work if we restart, or start a new one due to fail

Comment (strophy) [reply ansible/roles/elastic-beats/tasks/main.yml:96]: I wasn't able to get Tenderdash to write logs to files, how did we do this last time? With something like `nohup`?

I restarted a few containers during testing to change the logging verbosity and filebeat was still able to push logs to elasticsearch, why do you think this won't work?

File: ansible/roles/elastic-beats/tasks/main.yml [hunk 2]
```diff
@@ -89,6 +93,28 @@
...
+              - rename:
+                  fields:
+                    - from: "json._msg"
+                      to: "message"
+                  ignore_missing: true
+                  fail_on_error: true
+              - rename:
+                  fields:
+                    - from: "json.level"
+                    - to: "log.level"
```
Comment (shumkov) [inline ansible/roles/elastic-beats/tasks/main.yml:117]: Unify error level with Drive

Comment (strophy) [reply ansible/roles/elastic-beats/tasks/main.yml:117]: This seems to be consistent already. 
```
tenderdash: 'error', 'debug', 'info'
drive:      'fatal', 'error', 'warn', 'info', 'debug', 'trace', 'silent'
```

File: ansible/roles/elastic-stack/files/index-templates/logs-drive.tenderdash.json [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        },
+        "log.level": {
+          "type": "keyword",
+          "index": true,
+          "store": false
+        },
+        "message": {
+          "type": "text",
+          "index": true,
+          "store": true
```
Comment (shumkov) [inline ansible/roles/elastic-stack/files/index-templates/logs-drive.tenderdash.json:39]: Why do we store it?

**Review by shumkov — APPROVED** [review]
👍

**Pull request merged**